docs/application/native/guides/ui/dali/actors-n.md

```diff
@@ -21,13 +23,19 @@ An actor has the following concrete types:
 - **Camera actor** determines which view of the whole virtual 3D world is rendered on the screen. By default, it is best suited for 2D applications, so you do not need to care about it in this case. For more information, see the `Dali::CameraActor` class (in [mobile](../../../../../org.tizen.native.mobile.apireference/classDali_1_1CameraActor.html) and [wearable](../../../../../org.tizen.native.wearable.apireference/classDali_1_1CameraActor.html) applications).
 - **Layer** provides a mechanism for overlaying groups of actors on top of each other. For more information, see the `Dali::Layer` class (in [mobile](../../../../../org.tizen.native.mobile.apireference/classDali_1_1Layer.html) and [wearable](../../../../../org.tizen.native.wearable.apireference/classDali_1_1Layer.html) applications).
 
-## Actors and Stage
+## Adding an Actor to the Stage
+
+Stage is a top-level object that represents the entire screen.
+It is used for displaying a hierarchy of actors managed by the [scene graph](http://en.wikipedia.org/wiki/Scene_graph) structure, which means an actor inherits a position relative to its parent, and can be moved in relation to this point.
```

Comment:
It is used for displaying a hierarchy of actors managed by the [scene graph](http://en.wikipedia.org/wiki/Scene_graph) structure, which means an actor inherits a position relative to its parent, and can be moved in relation to this point.
<<The last sentence ...moved in relation to this point..is not clear, I have tried to rephrase, please let me know if its technically fine>>

Please rephrase to:

It is used for displaying a hierarchy of actors managed by the [scene graph](http://en.wikipedia.org/wiki/Scene_graph) structure. This implies that an actor inherits a position relative to its parent position and can be moved to another position with respect to its current position.